front/.gitignore

```diff
@@ -36,3 +36,6 @@ storybook-static/*
 .vscode/*
 
 cypress.env
+
+# Component library build for unit tests
+internals/jest/cl2-component-library.esm.js
```

Comment:
I've for now added this to `.gitignore` to avoid duplication. Whenever you run `npm install` in the main repo, this file will get copied. However there might be some situation where people run `npm run test` before they run `npm install`, in which case this won't work...

We could also consider not adding it to `.gitignore` and always having it checked it, but then the versions might get out of sync. We also might  consider running the copy script before `npm run test`. Let me know what you think